planner/core/src/main/java/seaclouds/planner/Optimizer.java

```diff
@@ -0,0 +1,236 @@
+/*
+ * Copyright 2015 Universita' di Pisa
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package seaclouds.planner;
+
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.Sets;
+import seaclouds.utils.toscamodel.*;
+import seaclouds.utils.toscamodel.basictypes.IValueInteger;
+
+import java.util.*;
+
+/**
+ * Created by pq on 26/04/2015.
+ */
+public class Optimizer {
```

Comment:
Is this class different from the existing Optimizer (https://github.com/SeaCloudsEU/SeaCloudsPlatform/blob/master/planner/optimizer/src/main/java/eu/seaclouds/platform/planner/optimizer/Optimizer.java) ?